packages/common/src/utils/decimal.ts

```diff
@@ -164,6 +164,78 @@ export const formatCurrency = (
   }
 }
 
+/**
+ * Formats a number with subscript notation for many leading zeros after decimal.
+ * For example: 0.000068352 → 0.0₄68352
+ *
+ * @param num - The number to format
+ * @param locale - Locale for number formatting (defaults to 'en-US')
+ * @returns Formatted string with subscript notation for leading zeros
+ */
+export const formatCurrencyWithSubscript = (
+  num: number,
+  locale: string = 'en-US'
+): string => {
+  if (num === 0) return '$0.00'
+
+  try {
+    const decimalPlaces = getCurrencyDecimalPlaces(num)
+    const formatted = new Intl.NumberFormat(locale, {
```

Comment:
what is `Intl`? can't use fixedDecimal?

Reply:
standard JS API, using instead of FixedDecimal because the use cases are seperate